pandas/core/indexes/base.py

```diff
@@ -4167,6 +4169,37 @@ def invalid_op(self, other=None):
 Index._add_comparison_methods()
 
 
+def dispatch_missing(op, left, right, result):
+    """
+    Fill nulls caused by division by zero, casting to a diffferent dtype
+    if necessary.
+
+    Parameters
+    ----------
+    op : function (operator.add, operator.div, ...)
+    left : object, usually Index
+    right : object
+    result : ndarray
+
+    Returns
+    -------
+    result : ndarray
```

Comment:
I think this function is better in ``missing``

Reply:
Sounds good.